problems/1431.py

```diff
@@ -0,0 +1,25 @@
+class Solution(object):
+    def kidsWithCandies(self, candies, extraCandies):
+        """
+        :type candies: List[int]
+        :type extraCandies: int
+        :rtype: List[bool]
+        """
+        #find the max number of candy
+        # for each element add extraCandies
+        maxNumberOfCandy = 0
+        for kid in candies:
+            maxNumberOfCandy = max(kid, maxNumberOfCandy)
+
+        output_list = [None]*len(candies)
```

Comment:
It is helpful to just append for interview/leetcode problems to make debugging easier

Reply:
But this is obviously the most clean solution so nice job!